### ISSUE TITLE
failed name validation error

### ISSUE THREAD

**Issue by rickcole3-yahoo** [Impetus/kundera#196]
I am seeing the following exception when I attempt to run a JPA query against my table. This query works fine when I run it as a native query, but the JPA version fails. This looks similar to another posted issue, but the error message is slightly different:

Caused by: javax.persistence.PersistenceException: InvalidRequestException(why:[6c6f676e616d65]=[636f6d2e67312e6463672e636f6d706f6e656e742e436173696e67] failed name validation (Not enough bytes to read value of component 0))
    at com.impetus.client.cassandra.thrift.ThriftClient.find(ThriftClient.java:914)
    at com.impetus.client.cassandra.query.CassQuery.populateEntities(CassQuery.java:143)
    at com.impetus.kundera.query.QueryImpl.getResultList(QueryImpl.java:155)
    at com.impetus.kundera.query.KunderaTypedQuery.getResultList(KunderaTypedQuery.java:249)

My table definition is as follows:

CREATE TABLE event_log (
  id text PRIMARY KEY,
  label text,
  logname text,
  lvl text
);
CREATE INDEX ON event_log(logname);

And here is my query code:

```
Query query = entityManager.createQuery("SELECT lc FROM LoggingConfiguration lc WHERE lc.logName = :logName", LoggingConfiguration.class);
query.setParameter("logName", logName);
List<Object> matchingConfigurations = query.getResultList();
```

Any idea on why this is happening?

Rick

**Comment by mevivs** [comment]
CQL and Thrift interoperability is an issue with Cassandra and we are planning to introduce CQL binary protocol support in Kundera 2.5. 

-Vivek

**Comment by rickcole3-yahoo** [comment]
Cool. Is there an ETA on 2.5?

**Comment by mevivs** [comment]
On 2.5 work is planned and we started working on it. Will post an update on Date.

-Vivek

**Comment by mevivs** [comment]
Already fixed and released with 2.5.1
